src/App.php

```diff
@@ -927,10 +927,10 @@ public function getTag($tag = null, $attr = null, $value = null)
 
         if ($value !== null) {
             $result = [];
-            foreach ((array)$value as $v) {
+            foreach ((array) $value as $v) {
                 if (is_array($v)) {
                     $result[] = $this->getTag(...$v);
-                } elseif (in_array($tag, ['script', 'style'])) {
+                } elseif (in_array($tag, ['script', 'style'], true)) {
```

Comment:
Hey, let fix this in develop immediatelly.

Reply:
review and approve and you can merge this in :)